### PR TITLE
Add FAQ section for Tasklist

### PR DESCRIPTION
**Pull request by vsgoulart**
Closes camunda-cloud/tasklist#1410

### CODE REVIEW

File: docs/components/tasklist/faq.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+id: faq
+title: FAQ
```
Comment (akeller) [inline docs/components/tasklist/faq.md:3]:
```suggestion
title: Tasklist FAQ
```

File: docs/components/tasklist/faq.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+description: "Frequently asked questions about Tasklist"
+---
+
+### Why my submit button is missing when I use Camunda Forms?
```
Comment (akeller) [inline docs/components/tasklist/faq.md:7]:
```suggestion
### Why is the submit button missing when I use Camunda Forms?
```

**Review by akeller — CHANGES_REQUESTED** [review]
Couple of small changes here.

Do you have other questions to add or are you looking to merge this with just the one?

**Comment by vsgoulart** [comment]
> Do you have other questions to add or are you looking to merge this with just the one?

@akeller I'll check with the team if they have anything else that they would like to include, I only created the FAQ section because a user was confused by this behaviour and I didn't know exactly where to put that information :sweat_smile:

**Review by santanaluiz — APPROVED** [review]

**Comment by vsgoulart** [comment]
We didn't have anything else relevant to add so I'll just merge with this one @akeller

**Review by akeller — APPROVED** [review]

**Pull request merged**